server/Dockerfile

```diff
@@ -1,5 +1,21 @@
+FROM curlimages/curl AS download
+WORKDIR /var/tmp
+RUN curl -f -L -O https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver \
+    -f -L -O https://hgdownload.cse.ucsc.edu/goldenpath/hg19/liftOver/hg19ToHg38.over.chain.gz \
+    -f -L -O https://hgdownload.cse.ucsc.edu/goldenpath/hg38/liftOver/hg38ToHg19.over.chain.gz && \
+    gunzip ./hg38ToHg19.over.chain.gz && \
+    gunzip ./hg19ToHg38.over.chain.gz && \
+    chmod +x ./liftOver
+
+# Development image 
+FROM node:16-slim AS dev
+WORKDIR /usr/src/app
```

Comment:
This directive isn't needed as it doesn't affect the commands in this stage and it's in the `base` stage